src/lib/services/CommandLineSupport.ts

```diff
@@ -77,10 +77,14 @@ export abstract class CommandLineSupport extends AsyncCreatable {
 
 			// When data is passed back up to us, pop it onto the appropriate string.
 			cp.stdout.on('data', data => {
-				stdout += data;
+				if (!this.handleLiveOut(String(data))) {
```

Comment:
I don't love this check.
If live output and at-the-end output use different start and end strings, couldn't we just pass all output through both at the appropriate times? The output processor would process real time output as it's received, and then it would process at-the-end output at the end, right?